admin/server/projects.go

```diff
@@ -132,8 +133,14 @@ func (s *Server) CreateProject(ctx context.Context, req *adminv1.CreateProjectRe
 		return nil, status.Error(codes.InvalidArgument, err.Error())
 	}
 
+	projectURL, err := url.JoinPath(s.opts.FrontendURL, fmt.Sprintf("-/%s/%s", org.Name, proj.Name))
+	if err != nil {
+		return nil, status.Error(codes.Internal, fmt.Sprintf("project url generation failed with error %s", err.Error()))
+	}
```

Comment:
1. `url.JoinPath` can take any number of args, so you don't need `fmt.Sprintf`
2. Another PR that'll be merged today removes the initial `/-`, so remove it here also